.github/workflows/ci-workflow.yml

```diff
@@ -4,39 +4,39 @@ on: [push, pull_request]
 
 jobs:
   lint_frontend:
-    defaults: 
+    defaults:
       run:
         working-directory: frontend
-    name: Check tsc, lint, and prettier on front end 
+    name: Check tsc, lint, and prettier on front end
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '21.x'
-    - run: npm install
-    - run: npm run lint
-    - run: npm run prettier
-      env:
-        CI: true
-  lint_backend: 
-    defaults: 
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "21.x"
```

Comment:
prettier wants double quote instead of single quotes now?

Reply:
maybe cuz its yml? I'm not sure, the checks do pass though